### PR TITLE
docs: update contributing guide

### PR DESCRIPTION
**Pull request by straker**
Closes issue: #2667

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,33 +11,29 @@ Submitting code to the project? Please review and follow our
 
 ### Code Quality
 
-Although we do not have official code style guidelines, we can and will request you to make changes
-if we think that your code is sloppy. You can take clues from the existing code base to see what we
-consider to be reasonable code quality. Please be prepared to make changes that we ask of you even
-if you might not agree with the request(s).
+Although we do not have official code style guidelines, we can and will request you to make changes if we think that your code is sloppy. You can take clues from the existing code base to see what we consider to be reasonable code quality. Please be prepared to make changes that we ask of you even if you might not agree with the request(s).
```
Comment (stephenmathieson) [inline CONTRIBUTING.md:14]: I do not think we should use the word "sloppy" here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,33 +11,29 @@ Submitting code to the project? Please review and follow our
...
 The JavaScript files in this project are formatted by [Prettier](https://prettier.io/). Additionally, we prefer:
 
-1. Tabs over spaces
-2. Single quotes for string literals
-3. Function definitions like `function functionName(arguments) {`
-4. Variable function definitions like `Class.prototype.functionName = function (arguments) {`
-5. Use of 'use strict'
-6. Variables declared at the top of functions
+1. Spaces over tabs
+1. Single quotes for string literals
```
Comment (stephenmathieson) [inline CONTRIBUTING.md:21]: Prettier does this for us. I don't think we need to spell this out.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -11,33 +11,29 @@ Submitting code to the project? Please review and follow our
...
+1. Function definitions like `function functionName(arguments) {`
```
Comment (stephenmathieson) [inline CONTRIBUTING.md:22]: Can we use [ESLint](https://eslint.org/docs/rules/func-style) for this?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -11,33 +11,29 @@ Submitting code to the project? Please review and follow our
...
+1. Using `const` for variables that do not change and `let` for ones that do
```
Comment (stephenmathieson) [inline CONTRIBUTING.md:23]: This too could be managed by [ESLint](https://eslint.org/docs/rules/prefer-const).

**Review by stephenmathieson — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (dylanb) [reply CONTRIBUTING.md:14]: It was being used before

Comment (stephenmathieson) [reply CONTRIBUTING.md:14]: That doesn't mean it was "nice" to begin with.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -11,16 +11,15 @@ Submitting code to the project? Please review and follow our
 
 ### Code Quality
 
-Although we do not have official code style guidelines, we can and will request you to make changes if we think that your code is sloppy. You can take clues from the existing code base to see what we consider to be reasonable code quality. Please be prepared to make changes that we ask of you even if you might not agree with the request(s).
+Although we do not have official code style guidelines, we can and will request you to make changes if we feel the changes are warranted. You can take clues from the existing code base to see what we consider to be reasonable code quality. Please be prepared to make changes that we ask of you even if you might not agree with the request(s).
```
Comment (stephenmathieson) [inline CONTRIBUTING.md:14]: Thank you 👍

**Review by stephenmathieson — APPROVED** [review]

**Pull request merged**